packages/react/src/types.ts

```diff
@@ -31,5 +31,65 @@ export type CssObject<TProps> = Readonly<{
 export type CssFunction<TProps = unknown> =
   | CssType<TProps>
   | BasicTemplateInterpolations // CSS values in tagged template expression
+  | null
   | boolean // Something like `false && styles`
   | undefined; // Something like `undefined && styles`
+
+// List of pseudo-classes and pseudo-elements are from csstype
+// but with & added in the front, so that we target the current element
+// (instead of a child element)
```

Comment:
```suggestion
// List of pseudo-classes and pseudo-elements are from csstype
// but with & added in the front. Compiled supports both &-ful
// and &-less forms and both will target the current element
// (`&:hover` <==> `:hover`), however we force the use of the
// &-ful form for consistency.
```

or something like this

Reply:
Sweet will add in